crypto/ocsp/ocsp_client.c

```diff
@@ -71,17 +81,19 @@ int OCSP_single_get0_status(OCSP_SINGLERESP *single, int *reason,
     OPENSSL_PUT_ERROR(OCSP, ERR_R_PASSED_NULL_PARAMETER);
     return -1;
   }
+  if(single->certStatus == NULL) {
+    OPENSSL_PUT_ERROR(OCSP, ERR_R_PASSED_NULL_PARAMETER);
+    return -1;
+  }
 
   OCSP_CERTSTATUS *cst = single->certStatus;
   int status = cst->type;
-  /*
-   * If certificate status is revoked, we look up certificate revocation
-   * time and reason
-   */
+
+  // If certificate status is revoked, we look up certificate revocation time and reason
   if (status == V_OCSP_CERTSTATUS_REVOKED) {
     OCSP_REVOKEDINFO *rev = cst->value.revoked;
     if (revtime != NULL) {
-      *revtime = rev->revocationTime;
+      OPENSSL_memcpy(revtime, &rev->revocationTime, sizeof(&rev->revocationTime));
```

Comment:
I'm not sure what you want to achieve with this memcpy, it is equivalent to `*revtime = rev->revocationTime`. My question in the previous review relating to this is: if you do `*revtime = rev->revocationTime` you will now have 2 pointers pointing to the same piece of memory (`*revtime`  and `rev->revocationTime`) -- is this a potential problem once you exit the function, i.e., is there a chance that in another function you use `revtime` to change something in the data while you don't want it to be changed in `rev->revocationTime`. Another potential issue if we are not careful is double freeing...

Reply:
Sorry, I didn't realize it was just the same pointer assignment. My original intention was to copy the values of `rev->revocationTime` over to `revtime` instead. 
That being said, the purpose of retrieving `revtime` is to extract the revocation time value from the der response sent back from the OCSP responser. The `revtime` value is extracted only to check if the timestamp is valid and is not intended to be changed in another function. 
I'll change it back to the original way for now.